packages/cloudflare/README.md

```diff
@@ -13,7 +13,13 @@ You can use [`create-next-app`](https://nextjs.org/docs/pages/api-reference/cli/
 - add the following `devDependencies` to the `package.json`:
 
   ```bash
+  npm add -D wrangler@latest @opennextjs/cloudflare
```

Comment:
Why `wrangler@latest` but not `@opennextjs/cloudflare@latest`?

Reply:
yeah I had the same exact thought!

I don't know, maybe because `@opennextjs/cloudflare` is still exeprimental?

I just kept this as it was already in the README 🤷